### ISSUE TITLE
Problem with new java/lang/String/StripIndent.java JTReg Test on JDK13

### ISSUE THREAD

**Issue by adam-thorpe** [eclipse/openj9#6878]
Failure link
------------
- https://ci.adoptopenjdk.net/view/Test_openjdk/job/Test_openjdk13_j9_sanity.openjdk_ppc64le_linux/36/console
- Version: 13+33
- Platforms: all
- Caused due to pull request from last night: https://github.com/eclipse/openj9/pull/6624

Optional info
-------------
- Intermittent failure: No
- Regression or new test: New Test

Failure output (captured from console output)
---------------------------------------------
```
06:00:04  "   abc
06:00:04     def
06:00:04  " not equal "   abc
06:00:04     def
06:00:04  "
06:00:04  java.lang.RuntimeException
06:00:04  	at StripIndent.verify(StripIndent.java:53)
06:00:04  	at StripIndent.test1(StripIndent.java:45)
06:00:04  	at StripIndent.main(StripIndent.java:33)
06:00:04  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
06:00:04  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
06:00:04  	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
06:00:04  	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
06:00:04  	at com.sun.javatest.regtest.agent.MainActionHelper$AgentVMRunnable.run(MainActionHelper.java:298)
06:00:04  	at java.base/java.lang.Thread.run(Thread.java:831)
```

**Comment by DanHeidinga** [comment]
fyi @fengxue-IS

**Comment by fengxue-IS** [comment]
looking into this

**Comment by fengxue-IS** [comment]
This is one of testcases that the Java13 API doc is not clear on the right behavior.
I have asked @andrew-m-leonard to confirm with openjdk on details of the method behavior.

In the mean while, I suggest we exclude this test until we received response from openjdk

**Comment by adam-thorpe** [comment]
Okay, I'll exclude this then

**Comment by fengxue-IS** [comment]
@adam-thorpe the test failure have been fix by #6952, please re-enable this test, thanks

**Comment by adam-thorpe** [comment]
Unexcluded: https://github.com/AdoptOpenJDK/openjdk-tests/pull/1331
